src/XcooBee/Store/CachedData.php

```diff
@@ -66,6 +69,10 @@ private static function _getDriver() {
      */
     public function setStore($key, $value)
     {
+        if ($this->_store === null) {
+            return null;
```

Comment:
Can you just return nothing in PHP? This is a setter method. Or is PHP like JavaScript where `undefined` is returned when just using the `return` statement without a value?
```suggestion
            return;
```
If yes, then maybe not return anything in the `clearStore` function below.

Reply:
I would prefer that we are explicit in our returns. Though it should be equivalent.